scripts/products/get-setProductTypes-txdata.js

```diff
@@ -4,11 +4,52 @@ const fs = require('fs');
 const { parse: csvParse } = require('csv-parse/sync');
 
 const ipfsClient = require('ipfs-http-client');
-const IPFS_API_URL = 'https://api.nexusmutual.io/ipfs-api/api/v0';
-const ipfs = ipfsClient({ url: IPFS_API_URL });
+
+const { simulateTransaction, constants } = require('./helpers');
 
 const { MaxUint256 } = ethers.constants;
-const COVER_ADDRESS = '0xcafeac0fF5dA0A2777d915531bfA6B29d282Ee62';
+const { COVER_ADDRESS, IPFS_API_URL } = constants;
+
+const ipfs = ipfsClient({ url: IPFS_API_URL });
+
+const verifyDecodedTxInputs = (csvProductTypeData, decodedTxInputs) => {
+  for (const csvProductTypes of csvProductTypeData) {
+    // Find matching object in decodedTxInputs by 'Product Name'
+    const productTypeName = csvProductTypes.Name;
+    console.log(productTypeName);
+    console.log(decodedTxInputs);
+    const decodedProductType = decodedTxInputs.find(decoded => decoded.productTypeName === productTypeName);
+
+    if (!decodedProductType) {
+      console.log(`\nVerification failed. No matching product found for: ${productTypeName}`);
+      return;
+    }
+
+    const { claimMethod, gracePeriod } = decodedProductType.productType;
+    //
+    // Set csv default values
+    const csvProductTypeId = csvProductTypes.Id || MaxUint256.toString();
+    const csvGracePeriod = csvProductTypes['Grace Period (days)'] * 24 * 3600; // This MUST be in seconds
```

Comment:
nit: should we explicitly convert to Number?

Reply:
It does it automatically when the operation is multiplication, but I'll add it just for cleaner code